packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -204,10 +217,15 @@ const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplicati
 });
 ```
 
+### Associate cross-region stack
```

Comment:
Just curious, if we do not support this feature, do we need to mention it here?

Reply:
I think it is needed since customer cdk app may contain cross region stacks, and introduction of ApplicationAssociator could not work as expected. This gives the clear expectation.

Reply:
make sense, explaining clearly is always good